src/main.scss

```diff
@@ -0,0 +1,10 @@
+.main {
+  width: 944px;
+  height: 862px;
```

Comment:
Avoid using fixed width and height for the layout. It's better to make it responsive by using percentages or CSS Grid/Flexbox properties that allow for more fluid scaling.